g_docs.go

```diff
@@ -557,10 +557,15 @@ func parserComments(comments *ast.CommentGroup, funcName, controllerName, pkgpat
 						para.Format = paraFormat
 					}
 				}
-				if len(p) > 4 {
+				switch len(p) {
+				case 5:
 					para.Required, _ = strconv.ParseBool(p[3])
 					para.Description = strings.Trim(p[4], `" `)
-				} else {
+				case 6:
+					para.Default = str2RealType(p[3], para.Type)
```

Comment:
@astaxie @xiaoqiang0 I get an error on this line: 
`./g_docs.go:565: para.Default undefined (type swagger.Parameter has no field or method Default)`

Am I missing something?